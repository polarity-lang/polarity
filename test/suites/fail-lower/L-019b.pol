infix _ -> _ := Fun(_,_,_)