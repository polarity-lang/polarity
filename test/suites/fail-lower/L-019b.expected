L-019

  × Invalid infix declaration: The right hand side of an infix declaration must take exactly two arguments.
   ╭─[L-019b.pol:1:17]
 1 │ infix _ -> _ := Fun(_,_,_)
   ·                 ──────────
   ╰────
